### ISSUE TITLE
Bed leveling bug, please update new branch of firmware 1.2.8

### ISSUE THREAD

**Issue by Sv3nergie** [ANYCUBIC-3D/ANYCUBIC_CHIRON#3]
adjustments to the individual probe points are not stored, when changes are made a fixed value is set and the whole bed leveling is messed up. When fixed, is it possible to update this github branch please.

**Comment by alexnv** [comment]
Figured out what is the problem with the sensor, in the firmware confused type of sensor contact, instead of normal-closed specified normally-open (or Vice versa)

**Comment by Sv3nergie** [comment]
No, my problem is not sensor related, thats ok. It's adjusting those points that don't work. It is fixed in the hex file you can flash from the anycubic site (version 1.2.8). But I like tmc drivers and need the 1.2.8 version in marlin code please. Can u please upload sources Anycubic ?

**Comment by jsprinceiii** [comment]
I also asked via support. Was told maybe soon. 
![anycubic-marlin-128_support](https://user-images.githubusercontent.com/26557171/47938074-e73dd680-deb0-11e8-9fde-93646cd81bf1.JPG)
